src/translations/en.json

```diff
@@ -254,8 +254,9 @@
   "claim.sn": "SN",
   "claim.edit.items.rejectall": "Reject All items",
   "claim.edit.items.delete": " ",
-  "claim.claimAttachment.type": "SelectAttachmentType",
```

Comment:
This is removed here. It is not used?

Reply:
It was defined two times:
`  "claim.claimAttachment.type": "Type",
`  "claim.claimAttachment.type": "SelectAttachmentType",

So I removed newer one, I think it was left from this old claim code that was migrated and then reverted.